src/core/vars/api.rb

```diff
@@ -38,8 +38,10 @@ def var!(var_name, **opts)
   # Since the features are defined just using variables, before calling this
   # method be sure that `setup_vars` has already been called.
   #
-  def has_feature?(feature_name)
-    get_var!("features", single_match: :force).include?(feature_name.to_s)
+  def has_feature?(*features_name)
+    features_name.all? do |feature_name|
```

Comment:
Better to transform every feature name before checking them:

```ruby
feature_names.map(&:to_s)
```

Note: using the block shortcut is very handy 😄